core/controllers/learner_goals.py

```diff
@@ -57,6 +57,12 @@ class LearnerGoalsHandler(
         'DELETE': {}
     }
 
+    # Note: Currently this handler only accepting one type of activity
+    # which is 'ACTIVITY_TYPE_LEARN_TOPIC', so 'activity_type' argument
+    # is not used inside the function body but, in future if this handler
```

Comment:
...inside the function body. But, in future, if this ...

Reply:
Done